USFMToolsSharp/HtmlRenderer.cs

```diff
@@ -118,14 +189,65 @@ private string RenderMarker(Marker input)
                     output.AppendLine(mTMarker.Title);
                     output.AppendLine("</div>");
                     break;
+                case FMarker fMarker:

```

Comment:
There still needs to be a marker rendered here to specify that there is a footnote.